src/python/zquantum/core/interfaces/ansatz_test.py

```diff
@@ -15,7 +14,7 @@ def test_set_number_of_layers_invalidates_parametrized_circuit(self, ansatz):
         # Given
         new_number_of_layers = 100
         if ansatz.supports_parametrized_circuits:
-            initial_circuit = ansatz.parametrized_circuit
+            ansatz.parametrized_circuit
```

Comment:
I prefer this syntax, as it's more readable to me
```suggestion
            _ = ansatz.parametrized_circuit
```

Reply:
I've removed the line in 71301c5, assuming that we shouldn't need to test property getters.